src/bolus/index.js

```diff
@@ -1,67 +1,199 @@
-const match = require('../match');
-const base = require('../base/index');
+const match = require("../match");
+const base = require("../base/index");
 
 class BolusScreen extends base.Screen {
-    constructor(language) {
-        super({
-            screenText: language.bolusScreen,
-            generalText: language.general,
-            header: {
-                backLabel: language.general.Cancel,
-            },
-            open: {
-                isBtn: true,
-                label: language.bolusScreen.Header,
-            },
-        });
-    }
-    get DoneButton() {
-        return match.accessible.Button(this.generalText.Done).atIndex(0);
-    }
-    get DeliverButton() {
-        return match.accessible.Button(this.screenText.Deliver);
-    }
-    get EnterBolusButton() {
-        return match.accessible.Button(this.screenText.EnterBolus);
-    }
-    get BolusLabel() {
-        return match.accessible.TextLabel(this.screenText.Header);
-    }
-    get BolusUnits() {
-        return match.accessible.TextLabel(this.screenText.Unit).atIndex(0);
-    }
-    get RecommendedLabel() {
-        return match.accessible.TextLabel(this.screenText.Recommended);
-    }
-    get RecommendedBolusLabel() {
-        return match.accessible.TextLabel(this.screenText.RecommendedBolus);
-    }
-    get RecommendedBolusUnits() {
-        return match.accessible.TextLabel(this.screenText.Unit).atIndex(1);
-    }
-    get ActiveCarbsLabel() {
-        return match.accessible.TextLabel(this.screenText.ActiveCarbs);
-    }
-    get BolusSummaryHeader() {
-        return match.accessible.TextLabel(this.screenText.BolusSummary);
-    }
-    get GlucoseHeader() {
-        return match.accessible.TextLabel(this.screenText.Glucose).atIndex(1);
-    }
+  constructor(language) {
+    super({
+      screenText: language.screenText,
+      generalText: language.generalText,
+      header: {
+        backLabel: language.generalText.Cancel,
+      },
+      open: {
+        isBtn: true,
+        label: language.screenText.Header,
+      },
+    });
+  }
+  get DoneButton() {
+    return match.accessible.Button(this.generalText.Done).atIndex(0);
+  }
+  get DeliverButton() {
+    return match.accessible.Button(this.screenText.Deliver);
+  }
+  get EnterBolusButton() {
+    return match.accessible.Button(this.screenText.EnterBolus);
+  }
+  get BolusLabel() {
+    return match.accessible.TextLabel(this.screenText.Header);
+  }
```

Comment:
i see that we are using match.accessible.Textlabel instead of match.label here. any eason?

Reply:
Likely this will change when I get to those tests. I haven't done a replace all, but changing as needed.